config/locales/en.yml

```diff
@@ -54,6 +54,7 @@ en:
         third_party_company_name : "Company name"
         third_party_name : "Full name of the representative"
         third_party_relationship: "Relationship to the subject"
+        third_party_email: "Email address (optional)"
```

Comment:
The value here will also be used in places like the error message (where you don't want the "optional" part to appear. There is also a translation for labels where this should be added.